Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift

```diff
@@ -15,30 +15,43 @@ struct TabTrayV2ControllerUX {
 }
 
 class TabTrayV2ViewController: UIViewController{
-    let tableView = UITableView()
+    // View Model
     lazy var viewModel = TabTrayV2ViewModel(viewController: self)
-    fileprivate let sectionHeaderIdentifier = "SectionHeader"
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        view.addSubview(tableView)
-        
+    // Views
+    lazy var tableView: UITableView = {
```

Comment:
Added tableview to its own closure variable view to avoid having heavy view did load method